src/trupti/CodingExam1/GetIndicesOfAddition.java

```diff
@@ -0,0 +1,52 @@
+package trupti.CodingExam1;
+
+import java.util.Scanner;
+
+public class GetIndicesOfAddition {
+	int [] getArray(Scanner scanner) {
+		
+		System.out.println("How many numbers you want to enter ?");
+		int size =scanner.nextInt();
+		int [] numberArray=new int[size];
+			for(int index=0;index<numberArray.length;index++) {
+				System.out.println("Enter Number"+(index+1)+" : ");
+				numberArray[index]=scanner.nextInt();
+			}
+			return numberArray;
+	}
+	
+	int [] getIndices(int [] numArray,int targetNum) {
+		int [] output=new int[2];
+		int tempSum=0;
+			for(int outerIndex=0;outerIndex<numArray.length;outerIndex++) {
+				for (int innerIndex=0;innerIndex<numArray.length;innerIndex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files